team-access.tf

```diff
@@ -42,3 +44,11 @@ resource "azurerm_key_vault_access_policy" "product_team_access_policy" {
     "Delete",
   ]
 }
+
+resource "azurerm_role_assignment" "this" {
+  count = var.enable_rbac_authorization == true ? 1 : 0
+
+  scope                = azurerm_key_vault.kv.id
+  role_definition_name = "Key Vault Administrator"
```

Comment:
hmm probably okay but I think needs a discussion as this adds the 'get' permission